src/main/java/com/foundation/salesforce/entities/Context.java

```diff
@@ -12,6 +12,8 @@
 
 package com.foundation.salesforce.entities;
 
+import io.restassured.response.Response;
+
 /**
  * Context class.
```

Comment:
In this comment you can please add more information about the class.

Reply:
done